docs/requirements.txt

```
@@ -1,5 +1,5 @@
 mkdocs==1.5.2
-mkdocs-material==9.2.4
-mkdocs-version-annotations==1.0.0
+mkdocs-material==9.1.15
```

Looks like the cookie is behind this app

```suggestion
mkdocs-material==9.2.4
```